### PR TITLE
Added help function and decode in memes.py

### PR DESCRIPTION
**Pull request by adnan007d**

### CODE REVIEW

**Comment by ricardoprins** [comment]
Hi!

Great job. We've had another PR changing the client.py. Could you handle these conflicts? 
Also, could you squash/rebase your PR so that you have a single commit?

**Comment by adnan007d** [comment]
Should I push the resolved conflict directly to the master branch or create a new branch ?

**Comment by gat786** [comment]
When you have conflicts what you do is pull the branch with which you are having a conflict and try to merge it with the local branch at your machine. You fix those issues on your machine. After they are fixed you push the code. No need to create another branch. [This](https://www.youtube.com/watch?v=Mfz8NQncwiQ&t=982s&ab_channel=ScottHanselman) video might help you understand it.

File: src/modules/help.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+def help():
+    return """
+commands
+    !help:\t Help menu
+    ping:\t Replies with pong
+    !memes:\t Fetchs a random meme from reddit
+<<<<<<< HEAD
```
Comment (ricardoprins) [inline src/modules/help.py:7]: These lines should be removed (the ones added during the merge process by git)

**Review by ricardoprins — CHANGES_REQUESTED** [review]
After you fix the help file, it would be great if you could squash/rebase all your commits into a single one.

File: src/modules/help.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    !events:\t Gets the list of events
+=======
```
Comment (ricardoprins) [inline src/modules/help.py:9]: lines 7, 9, and 11

**Review by ricardoprins — CHANGES_REQUESTED** [review]

**Comment by adnan007d** [comment]
Did the squash worked ? I had multiple conflicts while doing it

**Comment by ricardoprins** [comment]
It didn't - when it works, you'll only see one commit in your PR. Let me share some links that might help you :)

https://www.digitalocean.com/community/tutorials/how-to-rebase-and-update-a-pull-request
https://blog.carbonfive.com/always-squash-and-rebase-your-git-commits/

**Comment by adnan007d** [comment]
:smiley: Finally done!! It took me a while to figure it out and the article helped a lot

**Comment by gat786** [comment]
Lets goo!

**Comment by gat786** [comment]
For PR's I need to write some different github action this action fails everytime.

**Comment by gat786** [comment]
What I would suggest you Adnan is don't stage files to commit which you don't intend to commit. Right now, there are like 6 files changes out of which you worked on only 2-3 of them. Now benefit of staging only files you need to change is that you don't change much of the codebase until you need to.

**Comment by adnan007d** [comment]
Okay, then no need to staged  files like pipfile, piplock and all then config files which I didn't change manually

**Comment by gat786** [comment]
Yep remove them from the commits.

**Comment by adnan007d** [comment]
Is it failing because of this command ? 
python -m pipenv run pylint `ls -R|grep .py$|xargs`
in pylint.yml
python -m pipenv run pylint `find . | grep .py$|xargs` 
I think this one should work fine

**Comment by gat786** [comment]
I can still see the changes to workflow files in your commits and theres also a change to pipfile.lock.

**Comment by ricardoprins** [comment]
> I can still see the changes to workflow files in your commits and theres also a change to pipfile.lock.

I thought those were meant to be there, my bad!

**Comment by adnan007d** [comment]
:rocket: :tada:  Finally done!! :smiley:

**Comment by gat786** [comment]
Now your PR looks so clean. 👍🏼 These small tricks can help maintainers so much when applied in a good way.

**Comment by gat786** [comment]
Locally this build works. And Im accepting this PR. Thanks for contributing.

**Comment by gat786** [comment]
You are the first contributor outside of Tesseract Coding to contribute to Ric-Bot. Thanks for taking time and doing this.

**Comment by ricardoprins** [comment]
Amazing, thanks @adnan007d!

**Pull request merged**